ledger/testing/testGenesis.go

```diff
@@ -25,25 +25,25 @@ import (
 	"github.com/algorand/go-algorand/protocol"
 )
 
-// testGenesisCfg provides a configuration object for NewTestGenesis.
-type testGenesisCfg struct {
+// GenesisCfg provides a configuration object for NewTestGenesis.
+type GenesisCfg struct {
 	rewardsPoolAmount basics.MicroAlgos
+	OnlineCount       int
 }
 
 // TestGenesisOption provides functional options for testGenesisCfg.
-type TestGenesisOption func(*testGenesisCfg)
+type TestGenesisOption func(*GenesisCfg)
 
-// TestGenesisRewardsPoolSize configures the rewards pool size in the genesis block.
-func TestGenesisRewardsPoolSize(amount basics.MicroAlgos) TestGenesisOption {
-	return func(cfg *testGenesisCfg) { cfg.rewardsPoolAmount = amount }
-}
+// TurnOffRewards turns off the rewards pool for tests that are sensistive to
+// "surprise" balance changes.
+var TurnOffRewards = func(cfg *GenesisCfg) { cfg.rewardsPoolAmount = basics.MicroAlgos{Raw: 100_000} }
```

Comment:
Assume we have to keep the rewards pool present for legacy purposes...

Reply:
Yeah, I don't want to muck with that. I can certainly _imagine_ a project to purge it entirely, it would clean up a lot of stuff, but it would be a big chunk of work without user visible gain.